### ISSUE TITLE
Display comments with Task on a card

### ISSUE THREAD

**Issue by finnwood** [mwakerman/kanbanist#26]
It would be nice to be able to see comments along with the task on a card. If possible it could be collapsible.

**Comment by amadaira** [comment]
Yes!!! At first show the first lines of the comments of the first comment of task...
And please with rich format to have clickable links on those comments

You could then have on the card both the link to the todoist task, but some litle description in the view with possibly a link to another app (CRM or whatever)

**Comment by amadaira** [comment]
> Yes!!! At first show the first lines of the comments of the first comment of task...
> And please with rich format to have clickable links on those comments
> 
> You could then have on the card both the link to the todoist task, but some litle description in the view with possibly a link to another app (CRM or whatever)

I have a workaround, as you can add a link in the description of a task with the sintax [TEXT DESCRIPTION](URL) and is visible and clickable by kanbanist. You can also edit the task name in kanbanist and add a link with that sintax and syncs with todoist seamlesly! Great!!! Now I have tha kanban panels with the todoist tasks and a link to a external system (google drive, CRM or whatever)
